src/explore-education-statistics-common/src/modules/find-statistics/components/__tests__/DataBlockTabs.test.tsx

```diff
@@ -381,6 +383,78 @@ describe('DataBlockTabs', () => {
     });
   });
 
+  // EES-4902 react-leaflet now uses ESM so unable to test this currently
```

Comment:
As mentioned in another comment - I've created a little PR to your branch with the react-leaflet fix #5502 

We should now be able to update this test to work correctly.